content/platform/swarm-learning/home.md

```diff
@@ -0,0 +1,727 @@
+---
+title: HPE Swarm Learning
+version: 1.2.0
+description: "HPE Swarm Learning is a decentralized, privacy-preserving Machine
+  Learning framework. "
+image: /img/platforms/guid-e80d248e-e754-498e-99d6-67508092f779-high.jpg
+width: large
+priority: 0
+frontpage: false
+active: false
+---
+HPE Swarm Learning is a decentralized, privacy-preserving Machine Learning framework. This framework utilizes the computing power at, or near, the distributed data sources to run the Machine Learning algorithms that train the models. It uses the security of a blockchain platform to share learnings with peers in a safe and secure manner. In  HPE Swarm Learning, training of the model occurs at the edge, where data is most recent, and where prompt, data-driven decisions are mostly necessary. In this completely decentralized architecture, only the insights learned are shared with the collaborating ML peers, not the raw data. This tremendously enhances data security and privacy.
+
+- - -
+
+## Articles
+
+* **[2﻿021 Nature Paper - Swarm Learning for decentralized and confidential clinical machine learning](https://www.nature.com/articles/s41586-021-03583-3)**
+* **[HPE Labs - Swarm Learning: Turn your distributed data into competitive edge](https://www.hpe.com/psnow/doc/a50000344enw?jumpid=in_lit-psnow-red)**
+* **[Research gate - Swarm Learning as a privacy-preserving machine learning approach for disease classification](https://www.researchgate.net/publication/342495847_Swarm_Learning_as_a_privacy-preserving_machine_learning_approach_for_disease_classification)**
+
+- - -
+
+## Learn more about HPE Swarm Learning and its underlying technology
+
+**\-﻿ What's All the Buzz about Swarm Learning?**
+
+<!--\[if gte mso 9]><xml>
```

Comment:
Hi Arpan,

Could you please delete all the <xml> fragments from the page.
These xml fragments are typically the result of a copy/paste from a MS-Word document.
Please use the "CTRL+SHIFT+V" to  "paste as plain text" and prevent from getting the XML fragment code lines in the post.

I don't know if it could be the reason for the Pull Request process failure in Netlify and AWS Amplify.

If it is not clear just ping me and we can work together.

Thank you
Denis

Reply:
Corrected. Please check